openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.11/3-in-memory-channel.yaml

```diff
@@ -203,7 +232,7 @@ spec:
               fieldRef:
                 fieldPath: metadata.namespace
           - name: DISPATCHER_IMAGE
-            value: registry.ci.openshift.org/openshift/knative-v1.10:knative-eventing-channel-dispatcher
+            value: registry.ci.openshift.org/openshift/knative-v1.11:knative-eventing-channel-dispatcher
```

Comment:
@matzew should this be a "TO BE REPLACED" as well? How does this image replacement work?